### PR TITLE
moved common controls to ui shared

### PR DESCRIPTION
**Pull request by edewit**
## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

File: libs/ui-shared/package.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  "scripts": {
+    "build": "wireit",
+    "lint": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "command": "vite build",
+      "dependencies": [
+        "../../libs/keycloak-js:build"
+      ],
```
Comment (jonkoops) [inline libs/ui-shared/package.json:25]: I don't believe we have a dependency on `keycloak-js` here, let's remove this one.
```suggestion
```

File: libs/ui-shared/src/main.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export { ContinueCancelModal } from "./continue-cancel/ContinueCancelModel";
+export { SelectControl } from "./controls/SelectControl";
+export { TextControl } from "./controls/TextControl";
+export { KeycloakTextInput } from "./keycloak-text-input/KeycloakTextInput";
+export { AlertProvider, useAlerts } from "./alerts/Alerts";
```
Comment (jonkoops) [inline libs/ui-shared/src/main.ts:5]: We should also export the prop types here so they can be re-used.

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: libs/ui-shared/src/main.ts (hunk 2 above)
Comment (edewit) [reply libs/ui-shared/src/main.ts:5]: right

**Review by jonkoops — CHANGES_REQUESTED** [review]
@edewit looks like there are some build errors, can you take a look?

**Review by jonkoops — APPROVED** [review]

**Pull request merged**